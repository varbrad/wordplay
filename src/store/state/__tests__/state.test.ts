import { createInitialState, StoreState } from '..'

describe('createInitialState', () => {
  it('should match a snapshot if no arguments provided', () => {
    expect(createInitialState()).toMatchSnapshot()
  })

  it('should allow for custom properties if provided', () => {
    const racks: StoreState['racks'] = []
    const state = createInitialState({ racks })
    expect(state.racks).toEqual(racks)
  })
})
