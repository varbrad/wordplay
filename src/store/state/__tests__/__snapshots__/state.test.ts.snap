// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createInitialState should match a snapshot if no arguments provided 1`] = `
Object {
  "racks": Array [],
}
`;
